### PR TITLE
Add t007

### PR DESCRIPTION
**Pull request by GerbenRienk**
Added test-doc t007 group classes.

### CODE REVIEW

**Comment by chae-reliatec** [comment]
targets issue #71

File: docs/tests/t007.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# T007: subject group classes
+**Abstract:**  
+This documents contains the definitions for tests that concern the management of subject group classes. Study group classes are "groups of groups", that can be used to label and filter study subjects.
```
Comment (chae-reliatec) [inline docs/tests/t007.md]: should "Study group classes" be "Study subject group classes"?

File: docs/tests/t007.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+**description:**  
+This test focuses on removing a subject group class and the underlying groups.
+
+**prerequisite:**  
+You must be logged in to a study for which you have the data-manager-role and you must have successfully completed step **T007-01: create a new group class**.  
+
+**workflow:**    
+1. click on **Tasks - Study Setup - Build Study**
+1. click on the magnifier-icon of task **4 Create Subject Group Classes**
+1. in the screen **Manage All Groups in Study [Study Name]** enter the name of event you removed in **T007-01: create a new group classs**
```
Comment (chae-reliatec) [inline docs/tests/t007.md]: Do you mean "... enter the name of the event you created ..."?

File: docs/tests/t007.md (hunk 1 above)
Comment (GerbenRienk) [reply docs/tests/t007.md]: It's a bit strange, but in the form it actually says Study groups. Maybe we should change that eventually in the interface. For now I will change this text and only use "subject".

File: docs/tests/t007.md (hunk 2 above)
Comment (GerbenRienk) [reply docs/tests/t007.md]: You're right. I corrected it and committed a new version.

Comment (chae-reliatec) [reply docs/tests/t007.md]: I can't find the commit. Did you pushed the changes?

Comment (GerbenRienk) [reply docs/tests/t007.md]: I commited a new version just now, Nov 18.

**Pull request merged**